views/includes/page-templates/beneficial-owner-gov.html

```diff
@@ -38,6 +38,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
       {% set sanctionsListText = "Is it on the sanctions list?" %}
       {% set sanctionsListHintText = "This means that it is subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+      {% set sanctionsListNoText = "No, it is not on the Sanctions List" %}
```

Comment:
The `sanctionsListText` should be `Is it on the UK Sanctions List?` as for AC in your story

Reply:
Updated this to the precise text.